src/main/java/mate/academy/bookstore/service/impl/UserServiceImpl.java

```diff
@@ -17,6 +19,7 @@ public class UserServiceImpl implements UserService {
     private final UserRepository userRepository;
     private final UserMapper userMapper;
     private final PasswordEncoder passwordEncoder;
+    private final ShoppingCartRepository shoppingCartRepository;
 
     @Override
     public UserResponseDto register(UserRegistrationRequestDto requestDto) {
```

Comment:
We have 2 operations in method that modify DB state, better to use relevant annotation
```suggestion
    @Transactional
    public UserResponseDto register(UserRegistrationRequestDto requestDto) {
```